### PR TITLE
 Default to using app dot URLs

### PR DESCRIPTION
**Pull request by tylerlevine**
We have migrated all primary API traffic to the app.bitgo*.com TLDs, and
it's time to start using those domains by default in the SDK.

Users who continue to pass the deprecated environment names will not
need to upgrade, and these will now just be aliases for the equivalent
supported environment.

Ticket: BG-21412

### CODE REVIEW

File: modules/core/test/v2/unit/environments.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { TestBitGo } from '../../lib/test_bitgo';
+import { AliasEnvironments } from '../../../src/v2/environments';
+
+describe('Environments', () => {
+  it('should swap alias environments for supported environments', () => {
+    const aliasEnvs = ['production', 'msProd', 'msTest', 'msDev'];
```
Comment (LeoKotschenreuther) [inline modules/core/test/v2/unit/environments.ts:6]: You forgot `msLatest` here.

File: modules/core/src/v2/environments.ts [hunk 2]
```diff
@@ -193,30 +173,6 @@ export const Environments: Environments = {
...
-    stellarFederationServerUrl: 'https://app.bitgo-test.com/api/v2/txlm/federation',
-  }),
-  msDev: Object.assign({}, devBase, {
-    uri: 'https://app.bitgo-dev.com',
-    stellarFederationServerUrl: 'https://app.bitgo-dev.com/api/v2/txlm/federation',
-  }),
-  msLatest: Object.assign({}, devBase, {
-    uri: 'https://app.bitgo-latest.com',
-    stellarFederationServerUrl: 'https://app.bitgo-latest.com/api/v2/xlm/federation',
-  }),
```
Comment (twtaylorbitgo) [inline modules/core/src/v2/environments.ts:219]: Probably needs to keep: https://docs.google.com/document/d/1ef89ZmswlN7rVQMZfvbUhDG6w8XlkmF8PGyrJxzJVIg/edit

in mind

File: modules/core/test/v2/unit/environments.ts (hunk 1 above)
Comment (tylerlevine) [reply modules/core/test/v2/unit/environments.ts:6]: Fixed

**Comment by tylerlevine** [comment]
> overall LGTM - how did `package-lock.json` sneak back in?

It was added in https://github.com/BitGo/BitGoJS/commit/b29911a1fe67f1262feaba779acab2cb02ac922c

**Pull request merged**